packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -0,0 +1,269 @@
+     * - `"small"`, matching `sizing.size_160`.
+     * - `"medium"`, matching `sizing.size_240`.
+     *
+     * Default: `"size_160"`
+     */
+    paddingSize?: "none" | "small" | "medium";
+    /**
+     * The box-shadow for the card, as a string identifier that matches a sizing token.
+     * This can be one of:
+     * - `"none"`: no elevation.
+     * - `"low"`, matching `boxShadow.low`.
+     *
+     * Default: `"none"`
+     */
+    elevation?: "none" | "low";
+};
+
+type Props = BaseCardProps & TagProps & DismissProps;
+/**
+ * The Card component is a flexible, reusable UI building block designed to
+ * encapsulate content within a structured, visually distinct container.
+ * Its primary goal is to present grouped or related information in a way that
+ * is visually consistent, easily scannable, and modular across different
+ * parts of the application.
+ *
+ * Cards provide a defined surface area with clear visual boundaries
+ * (via border-radius and box-shadow elevation tokens), making them ideal for
+ * use cases that involve displaying comparable content items side-by-side or
+ * in structured layouts such as grids, lists, or dashboards.
+ *
+ * Note: cards do not set a default width. Width styles should be set by the consumer
+ * with the `styles.root` prop, or a parent flex or grid container.
+ *
+ * ### Usage
+ *
+ * ```jsx
+ * import {Card} from "@khanacademy/wonder-blocks-card";
+ *
+ * <Card>
+ *   <Heading>This is a basic card.</Heading>
+ * </Card>
+ * ```
+ *
+ * ### Styling
+ *
+ * Cards can be customized via the following props:
+ *
+ * **`background` prop**
+ *
+ * | value | resolves to |
+ * |---|---|
+ * | `base-subtle` | `semanticColor.core.background.base.subtle` (light gray) |
+ * | `base-default` | `semanticColor.core.background.base.default` (white) |
+ *
+ * **`borderRadius` prop**
+ *
+ * | value | resolves to |
+ * |---|---|
+ * | `small` | `border.radius.radius_080` |
+ * | `medium` | `border.radius.radius_120` |
+ *
+ * **`elevation` prop**
+ *
+ * | value | resolves to |
+ * |---|---|
+ * | `none` | `none` |
+ * | `low` | `boxShadow.low` |
+ *
+ * **`paddingSize` prop**
+ *
+ * | value | resolves to |
+ * |---|---|
+ * | `none` | `sizing.size_0` |
+ * | `small` | `sizing.size_160` |
+ * | `medium` | `sizing.size_240` |
+ *
+ * Other styles can be applied via the `styles` prop, which accepts an object with `root` and `dismissButton` keys for custom styling of the card container and dismiss button respectively.
+ *
+ * ### Accessibility
+ * - When using `tag="section"` or `tag="figure"`, the `labels.cardAriaLabel` prop is required to provide an accessible name for the card.
+ * - When the `onDismiss` prop is provided, a dismiss button will be rendered. In this case, the `labels.dismissButtonAriaLabel` prop is required to provide an accessible label for the dismiss button.
+ * - Cards cannot be interactive elements themselves (e.g. buttons or links), but they can contain interactive elements as children.
+ */
```

Comment:
What do you think about moving these docs to a story description? Since the component documentation is long, the props table is no longer near the top! 
https://5e1bf4b385e3fb0020b7073c-uhnlrtvxlk.chromatic.com/?path=/docs/packages-card-card--docs

Reply:
Hm, I'm not sure what this meant exactly! Maybe it was referring to a specific story? Maybe there could be an `Overview.mdx` file or something more editorial? I think I'll opt to leave it for this PR and revisit in an update.